frontend/src/components/Toast.tsx

```diff
@@ -0,0 +1,42 @@
+import { useToast } from "@chakra-ui/react";
+import { useEffect, useState } from "react";
+
+type ToastStatus = "info" | "warning" | "success" | "error" | undefined;
+
+type ToastProps = {
+  status: ToastStatus;
+  title: string;
+  message: string;
+};
+
+const useToasts = (): ((
```

Comment:
Could you add a docstring header here. Also could you add some brief documentation throughout this file on how to use this hook?